### ISSUE TITLE
Hide apps: main / work profile

### ISSUE THREAD

**Issue by JonnyTech** [OpenLauncherTeam/openlauncher#586]
#### General Information

* **OpenLauncher Version:** 0.7.3 (41)
* **Android Version:** 9.0
* **Custom ROM:** AOSP GSI
* **Install Source:** F-Droid

Package: com.benny.openlauncher
Version: v0.7.3 (41)
Flavor: Default (release)
Build date: 2020-04-30T01:56Z
ISource: com.google.android.packageinstaller
VCS Hash: 73968b61ee99b74b4b8a7205c34d3bc8a58422d8

#### Description

Apps cloned to work profile now show in the drawer - many thanks. But if you hide either the main or work app then both disappear from the drawer. Is it possible to hide one or the other but not both?

**Comment by lost-geographer** [comment]
Are you talking about this issue : https://github.com/OpenLauncherTeam/openlauncher/issues/294 ?

**Comment by JonnyTech** [comment]
No, that talks about the home screen, my issue is with the **app drawer**. If I install an app to the main profile and clone it to the work profile then two apps show up in the drawer. This is correct. But if I go to settings and hide **either** the app in the main profile **or** the one in the work profile then **both** are hidden. This is incorrect. There should be still one shortcut remaining in the app drawer.
